numl/NeutrinoML/HDF5Maker_module.cc

```diff
@@ -145,9 +156,9 @@ void HDF5Maker::analyze(art::Event const& e)
   // Get MC truth
   art::Handle< vector< MCTruth > > truthHandle;
   e.getByLabel(fTruthLabel, truthHandle);
-  if (!truthHandle.isValid() || truthHandle->size() != 1) {
+  if (!truthHandle.isValid() || truthHandle->size() == 0) {
     throw art::Exception(art::errors::LogicError)
-      << "Expected to find exactly one MC truth object!";
+      << "Expected to find at least one MC truth object!";
```

Comment:
if there's more than one MC truth object, is getting the first one in the list and ignoring the others the right behaviour here? under what circumstances are there multiple truth objects? should we loop over all the available truths and add them to the table, and add another column for truth indices so they can be disambiguated later? if this is valid behaviour, we should split out the event table into an event table and a truth table, so we can store multiple true neutrino energies etc per event